code/game/gamemodes/changeling/powers/revive.dm

```diff
@@ -47,8 +48,9 @@
 			O.status = 0
 			O.trace_chemicals.Cut()
 		for(var/obj/item/organ/internal/IO in H.internal_organs)
-			IO.rejuvenate()
-			IO.trace_chemicals.Cut()
+			if(IO.status != ORGAN_ROBOT)
+				IO.rejuvenate()
+				IO.trace_chemicals.Cut()
```

Comment:
status is a bitfield, comparing it to a bit value with `!=` is gonna raise suspicion.
I understand that it *is* what you want to do (check that it's robotic, and pristine - unbroken and stuff), but that needs at least a comment. 
```suggestion
			if(IO.status == ORGAN_ROBOT)
				// If it's a robotic organ and no other flags are present (i.e. it's not broken/splintered etc) - leave it be.
				continue
			IO.rejuvenate()
			IO.trace_chemicals.Cut()
```